src/plugin-slots/PageBannerSlot/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from 'react';
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import { PageBanner } from '@openedx/paragon';
+
+export interface PageBannerSlotProps {
+  show: boolean;
+  dismissible: boolean;
+  onDismiss: () => void;
+  className: string;
+  children: ReactNode;
+  variant?: string;
+  dismissAltText?: string;
+}
+
+const PageBannerSlot: React.FC<PageBannerSlotProps> = ({
+  show,
+  dismissible,
+  onDismiss,
+  className,
+  children,
+  variant = 'info',
+  dismissAltText = 'Dismiss',
```

Comment:
This word "Dismiss" won't be translated, so we need another way of specifying it.